src/NServiceBus.Core/EndpointConfiguration.cs

```diff
@@ -21,14 +20,19 @@ namespace NServiceBus
     /// <summary>
     /// Configuration used to create an endpoint instance.
     /// </summary>
-    public class EndpointConfiguration : ExposeSettings
+    public  class EndpointConfiguration : ExposeSettings
```

Comment:
this class has so much space it should be named BusinessClass ![](http://i.imgur.com/oEhVvjZ.gif)

sorry, that was so bad.